examples/javascript/FeatureExtractor/FeatureExtractor_Image_Regression/sketch.js

```diff
@@ -71,7 +71,7 @@ function setupButtons() {
   // When the Dog button is pressed, add the current frame
   // from the video with a label of "dog" to the classifier
   document.querySelector("#addSample").addEventListener("click", function() {
-    regressor.addImage(slider.value);
+    regressor.addImage(parseFloat(slider.value));
```

Comment:
Should we handle this internally in ml5? I don't feel strongly but we could do this conversion there since passing in a string label is not compatible with regression!

Reply:
I was wondering about this too. The best logic I could consider is changing any values that seem numeric into a numeric value instead of a string label. 

Are string labels really not compatible? It seems like there might be some code in the model for associating string levels to some of the regression outputs. I'll open an issue to come back to this!

Reply:
#1092 https://github.com/ml5js/ml5-library/issues/1092

Reply:
Great question! The concept of regression is quite broad, but in the case of ml5 and the way `ml5.neuralNetwork()` was designed, the only valid outputs are numeric. If it's a string then by definition (at least in the case of ml5) it's a classification. I'm open to rethinking this of course!